### PR TITLE
feat: Prisma2 Beta 1 support + transition package name

### PR DESCRIPTION
**Pull request by Weakky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,10 +27,10 @@
     "postinstall": "node ./scripts/postinstall.js"
   },
   "peerDependencies": {
-    "nexus-future": "^0.13.0-next.11"
+    "nexus": "0.20.0-next.1"
```
Comment (jasonkuhrt) [inline package.json]: Peer dep should be removed, can be another, might as well now

File: package.json [hunk 2]
```diff
@@ -27,10 +27,10 @@
...
   },
   "dependencies": {
-    "@nexus/schema": "^0.12.0-rc.13",
+    "@nexus/schema": "0.12.0-rc.13",
```
Comment (jasonkuhrt) [inline package.json]:
```suggestion
    "@nexus/schema": "^0.13.0",
```

File: test/e2e/sqlite.test.ts [hunk 3]
```diff
@@ -1,32 +1,34 @@
...
-    nexusVersion,
+  const createAppResult = await ctx.npxNexusCreateApp(
+    {
+      packageManagerType: 'npm',
+      databaseType: 'SQLite',
+      nexusVersion,
+    },
     (data, proc) => {
-      if (stripAnsi(data).includes('server:listening')) {
+      if (stripAnsi(data).includes('server listening')) {
```
Comment (jasonkuhrt) [inline test/e2e/sqlite.test.ts:25]:
```suggestion
      if (data.includes('server listening')) {
```

File: package.json (hunk 2 above)
Comment (Weakky) [reply package.json]: Weird, I think you reviewed on an older commit 🤔. It is `0.13.0` already

**Pull request merged**